terrain_test.py

```
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""
+Visual test for the terrain generation.
+"""
+
+import curses
+import time
+from game.entities import Terrain
+
+
+def main(stdscr):
```

Ruff (0.8.2)

`main` is too complex (11 > 10)

(C901)